package.json

```diff
@@ -43,7 +43,6 @@
     "@types/lodash": "^4.14.170",
     "@types/react": "18.2.23",
     "@types/react-dom": "18.2.8",
-    "@types/react-transition-group": "4.4.8",
```

Comment:
why are you removing this? is it because its a devDep?

Reply:
We're also installing this within individual packages, and having it here could cause some version discrepancies